src/pages/kb/user-guide/integrations-and-api/slack-bot-posting-visualizations-to-slack.md

```diff
@@ -6,6 +6,9 @@ title: Slack Bot (posting visualizations to Slack)
 slug: slack-bot-posting-visualizations-to-slack
 ---
 
+{% callout danger %}
+The Redash Slackbot is now deprecated.
+{% endcallout %}
```

Comment:
Should we also remove the rest of the content on this page? And maybe link to the future forum post?

Reply:
Yes linking to the forum post is the right move. I'm drafting that in gdocs now. Happy to remove the rest of the page contents as well.

Reply:
Removed in 9a20101e14e3ad4e6f9c2158097f808310b0b026